tools/tsp-client/src/gh.ts

```diff
@@ -0,0 +1,97 @@
+import { execSync, spawn } from "child_process";
```

Comment:
nit the file name should be `git.ts` I think since `gh` makes me think of github and this isn't really github specific